examples/binaries/reserve-gas/src/lib.rs

```diff
@@ -66,7 +66,7 @@ pub type BlockCount = u32;
 mod wasm {
     use super::*;
     use gstd::{
```

Comment:
What about addition in gstd `pub use errors::Error` to be able to use `gstd::Error`

Reply:
Same for `gstd::Result`

Reply:
in separate pr sure